### PR TITLE
dprint on doc src files

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

Runs dprint on the output files with:
`yarn dprint`

```
$ dprint fmt
✨  Done in 0.18s.
```
## Notes to reviewers
I'm going to add Loren to this PR to review the dprint rules: `dprint.json`

### CODE REVIEW

**Comment by lorensr** [comment]
Can you do bracket spacing and single quote for TS?

image

Then if we regularly ran dprint, we wouldn't need #1596

File: docs-src/typescript/how-to-define-activity-return-values-in-typescript.md [hunk 1]
```diff
@@ -11,7 +11,17 @@ tags:
 
 In TypeScript, the return value is always a Promise.
 
+<<<<<<< HEAD
```
Comment (vkarpov15) [inline docs-src/typescript/how-to-define-activity-return-values-in-typescript.md:14]: Merge conflict accidentally made it in here?

File: docs-src/typescript/how-to-send-a-signal-with-start-in-typescript.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ tags:
 
 ```typescript
 import {WorkflowClient} from "@temporalio/client";
-import {yourWorkflow, joinSignal} from "./workflows";
+import {joinSignal, yourWorkflow} from "./workflows";
```
Comment (vkarpov15) [inline docs-src/typescript/how-to-send-a-signal-with-start-in-typescript.md:16]: One of the primary purposes of #1596 was to add spaces around the curly braces in TS import statements, which lines up with how the TS SDK docs' style. So this should be `import { joinSignal, yourWorkflow } from "./workflows'";`

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
@lorensr @vkarpov15 - any concerns?
I think we are hoping to go all in on dPrint now - and remove prettier

For the time being lets use dPrint locally - and then aim to set up a Github action if we need to?

**Pull request merged**